docs/CI.md

```
@@ -1,20 +1,491 @@
+    multicast-info-*-->CI-Tests.yml;
+    CI-Tests.yml-->COVERAGE;
+    CI-Tests.yml-->\*-Test-Report-\*-*;
+    CI-Tests.yml-->Integration-Summary-Artifact.txt;
+    multicast-build-*-->CI-DOCS.yml;
+    multicast-info-*-->CI-DOCS.yml;
+    CI-DOCS.yml-->Multicast-Documentation-*-ALL;
+    CI-DOCS.yml-->DOCUMENTATION-Summary-Artifact.txt;
+```
+
+1. **PUSH-EVENT**: This event triggers the entire Acceptance Testing CI/CD workflow.
+1. [CI-Build.yml](https://github.com/reactive-firewall/multicast/tree/HEAD/.github/workflows/CI-BUILD.yml):
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 2; Actual: 1; Style: 1/2/3

(MD029, ol-prefix)